src/Elements/Bundle/AlternateObjectTreesBundle/LevelDefinition/Input.php

```diff
@@ -52,15 +52,19 @@ public function getGroupedValues($condition, $offset = null, $limit = null)
         // create condition
         $where = $this->condition == '' ? 1 : $this->condition;
         if ($condition) {
+            $where = 'LIKE \'%'.$where.'%\'';
             $where .= ' AND ' . $condition;
```

Comment:
Have you tried to use double quotes you can do things like $where .= " AND $condition" or $where = "LIKE '%$where%'"

Reply:
yeah that looks even cleaner

Reply:
changed it